openshift-kube-apiserver/admission/autoscaling/managementcpusoverride/admission.go

```diff
@@ -217,6 +217,11 @@ func (a *managementCPUsOverride) Admit(ctx context.Context, attr admission.Attri
 		return err
 	}
 
+	if _, found := ns.Annotations[namespaceAllowedAnnotation]; !found && len(workloadType) > 0 {
+		pod.Annotations[workloadAdmissionWarning] = "skip pod CPUs requests modifications because namespace is not annotated for workloads"
```

Comment:
I think you should be more specific (keywords that allow googling the docs) in the message. I am not sure if the text clearly points at the workload partitioning feature. Think about people hearing about it for the first time.

Reply:
Thats a good point, updated language with a bit more detail to help with that